src/lucky_flow/spec.cr

```diff
@@ -0,0 +1,17 @@
+module LuckyFlow::Spec
+  macro setup
+    Spec.around_each do |spec|
+      if driver_name = (spec.example.all_tags & LuckyFlow::Registry.available).first?
```

Comment:
In Rspec, you can pass key-value pairs in `it`'s but in Crystal you only have tags which are a list of strings. So instead of requiring passing something like `driver: :chrome`, you have to do `tags: "chrome"` instead and then we take the overlap of the tags you passed with the available browsers you can use. If for some reason you do `tags: ["headless_chrome", "chrome"]`, there'd be an overlap of 2 but we take the first